resources/nl-core/nl-core-LaboratoryTestResult.xml

```diff
@@ -83,7 +82,6 @@
         <code value="Extension" />
         <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-CopyIndicator" />
       </type>
-      <isModifier value="false" />

```

Comment:
Eens met de wijziging, maar nu we officieel betapackages uitbrengen moet dit denk ik wel apart in een ticket worden doorgevoerd.

Reply:
Wijziging wordt opgepakt in ticket MM-4128.